content/docs/2.8/scalers/aws-dynamodb-stream.md

```diff
@@ -0,0 +1,149 @@
++++
+title = "AWS DynamoDB Streams"
+availability = "v2.8+"
+maintainer = "Community"
+description = "Scale applications based on AWS DynamoDB Streams"
+layout = "scaler"
+go_file = "aws_dynamodb_stream_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `aws-dynamodb-stream` trigger that scales based on the shard count of AWS DynamoDB Streams.
+
+```yaml
+triggers:
+- type: aws-dynamodb-stream
```

Comment:
Big nit - The scaler is called `AWS DynamoDB Streams` but our type is `aws-dynamodb-stream`.
Should we align them?

Reply:
@tomkerkhove 
let me align them to `aws-dynamodb-streams` and `AWS DynamoDB Streams` as the official service name looks DynamoDB Streams ([ref](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html)). I'll modify shortly. thank you!

Reply:
remember to update also this docs 🙏